server/src/collections/collections.service.ts

```diff
@@ -223,13 +230,26 @@ export class CollectionsService {
     if (res.data.length > 0) {
       for (const item of res.data) {
         const { id, name } = item;
-        const collectionStatistics = await this.getCollectionStatistics({
-          collection_name: name,
-        });
+
+        let count: number | string;
+
+        try {
+          const countRes = await this.count({
+            collection_name: name,
+          });
+          count = countRes.data;
+        } catch (error) {
+          const collectionStatisticsRes = await this.getCollectionStatistics({
+            collection_name: name,
+          });
+          count = collectionStatisticsRes.data.row_count;
```

Comment:
Can we get the exact error code?

Reply:
they are formatting the error code. I don't want to use the error for now.